pages/storage_and_backup/object_storage/cold_archive_overview/guide.de-de.md

```diff
@@ -1,9 +1,15 @@
 ---
 title: Cold Archive - Überblick (EN)
 excerpt: Discover the service, understand the capabilities and billing
-updated: 2023-01-29
+updated: 2023-12-20
 ---
 
+<style>
+.w-100 {
+    width:100% !important;
+}
+</style>
+
 ## Objective
 
 OVHcloud Cold Archive is  a class of storage designed for a long-term strage of non-actively used data.
```

Comment:
I would use "rarely accessed" data instead

Reply:
agreed, and there is also a small typo on "strage" -> storage. Thanks